hy/compiler.py

```diff
@@ -1139,6 +1141,14 @@ def compile_yield_expression(self, expr):
         node = asty.Yield if expr[0] == "yield" else asty.YieldFrom
         return ret + node(expr, value=ret.force_expr)
 
+    @builds("await", iff=PY35)
+    @checkargs(max=1)
+    def compile_await_expression(self, expr):
+        ret = Result(contains_yield=(not PY3))
```

Comment:
Maintaining `contains_yield` is not necessary here, since the method can only trigger in Python 3.